### PR TITLE
Quiz table

### PR DESCRIPTION
**Pull request by cheemurakami**
Added Quiz table and CRUD functionality.
Quiz is displayed in grammar show page.

Screen Shot 2021-01-26 at 8 15 36 PM
Screen Shot 2021-01-26 at 8 16 11 PM
Screen Shot 2021-01-26 at 8 16 23 PM
Screen Shot 2021-01-26 at 8 16 37 PM

Screen Shot 2021-01-27 at 3 03 02 PM

### CODE REVIEW

File: app/views/quizzes/_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <%= form.hidden_field :grammar_id %>
+
+  <div class="field">
+    <%= form.label :tokyo %>
+    <%= form.text_field :tokyo %>
```
Comment (gdomingu) [inline app/views/quizzes/_form.html.erb:18]: Could you give these a placeholder so we can see an example when we are creating these?

File: db/migrate/20210127001712_create_quizzes.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class CreateQuizzes < ActiveRecord::Migration[6.0]
+  def change
+    create_table :quizzes do |t|
+      t.text :tokyo
+      t.text :answer
+      t.references :grammar, null: false, foreign_key: true
```
Comment (gdomingu) [inline db/migrate/20210127001712_create_quizzes.rb:6]: Nice~

**Review by gdomingu — APPROVED** [review]
Great job! Just one small thing you feel like it. Otherwise FANTASTIC job! :shipit:

File: app/views/quizzes/_form.html.erb (hunk 1 above)
Comment (cheemurakami) [reply app/views/quizzes/_form.html.erb]: 👀 👂 

Screen Shot 2021-01-27 at 4 28 22 PM

**Review by gdomingu — APPROVED** [review]

**Pull request merged**